### ISSUE TITLE
Filter button not working

### ISSUE THREAD

**Issue by epikurus** [kokernutz/traktor-kontrol-screens#12]
After last update: 

 - Filter button does not work, it does not enable/disable filter in Traktor. 

 - Shift + Filter does not change between MixerFX banks.

**Comment by jkudish** [comment]
Same issue here

**Comment by kokernutz** [comment]
Is this on the S8? I will revert the change for the S8 as I don’t have it right.

**Comment by gcwioro** [comment]
just downloaded and it worked like a charm on my s5.
Best Mod I've ever seen.
Many Thanks.
BTW: To control the 3, 4 fx units would be a cool feature

**Comment by epikurus** [comment]
> Is this on the S8? I will revert the change for the S8 as I don’t have it right.

Yes, it happens on the S8. I'd be happy to test any changes to make it work.

Great work on the mod BTW :)

**Comment by kokernutz** [comment]
I reverted the S8 files back to their original state. @epikurus I may ask you to test a new file to see if I get the code right for the S8.

**Comment by epikurus** [comment]
@kokernutz of course! I'm using the mod on live situations FYI.

Just updated to the latest commit and the Filter button is working again.

**Comment by jkudish** [comment]
Thanks! Happy to help test on the s8 as well

**Comment by bascurtiz** [comment]
Awesome update @kokernutz !
But indeed, "Cycle through MixerFX on S5/S8 by pressing SHIFT-FILTER" doesn't work.
I assume when holding SHIFT and rotating a FILTER-knob it should select a different Filter setting.

Since I got a S8, I'm willing to help out.
Which part/file is responsible for this function?
And which values I could try to change?

Also, Midi Controls are Enabled in Traktor Pro 3 in Preferences.
But for this, it uses HID right?

**Comment by MrPatben8** [comment]
Found a workaround for this problem, here is the issue page for it.
https://github.com/kokernutz/traktor-kontrol-screens/issues/14
